src/app/MessageDef/CommandPath.h

```diff
@@ -44,6 +44,16 @@ enum
     kCsTag_CommandId  = 3,
 };
 
+struct Type
```

Comment:
The word Type is just too overloaded to be intuitive here.   I suppose we are saying that this is the 'type' of the path.  But I think it would be a lot clearer to just call this 'Path' or similar.